### PR TITLE
Adding an offline test

### PR DESCRIPTION
**Pull request by prateekbh**
**What kind of change does this PR introduce?**

Adds tests

**Did you add tests for your changes?**
Yes

**Summary**
Adds a test that our service workers actually work.

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by prateekbh** [comment]
I have no idea what i did there, but the tests seems to be flaky now

**Comment by prateekbh** [comment]
So a single test of `watch.test.js` works

**Comment by prateekbh** [comment]
Also is fine without `service-worker.test.js`

File: packages/cli/tests/service-worker.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		dir = join(dir, 'build');
+		server = getServer(dir);
+	});
+
+	afterAll(async () => {
+		await server.server.stop();
+		await browser.close();
+	});
+
+	it('builds the default output', async () => {
```
Comment (ForsakenHarmony) [inline packages/cli/tests/service-worker.test.js]: I guess this name isn't right 👀

Comment (prateekbh) [reply packages/cli/tests/service-worker.test.js]: lol fixing it

**Pull request merged**